examples/quickstart_xgboost_horizontal/code_horizontal.ipynb

```diff
@@ -29,32 +29,52 @@
+    "if not os.path.exists(CLASSIFICATION_PATH):\n",
+    "  os.makedirs(CLASSIFICATION_PATH)\n",
+    "  urllib.request.urlretrieve(\"https://www.csie.ntu.edu.tw/~cjlin/libsvmtools/datasets/binary/cod-rna\",\n",
+    "                             f\"{os.path.join(CLASSIFICATION_PATH, 'cod-rna')}\")\n",
+    "  urllib.request.urlretrieve(\"https://www.csie.ntu.edu.tw/~cjlin/libsvmtools/datasets/binary/cod-rna.t\",\n",
+    "                             f\"{os.path.join(CLASSIFICATION_PATH, 'cod-rna.t')}\")\n",
+    "  urllib.request.urlretrieve(\"https://www.csie.ntu.edu.tw/~cjlin/libsvmtools/datasets/binary/cod-rna.r\",\n",
+    "                             f\"{os.path.join(CLASSIFICATION_PATH, 'cod-rna.r')}\")\n",
+    "  urllib.request.urlretrieve(\"https://www.csie.ntu.edu.tw/~cjlin/libsvmtools/datasets/binary/ijcnn1.t.bz2\",\n",
+    "                             f\"{os.path.join(CLASSIFICATION_PATH, 'ijcnn1.t.bz2')}\")\n",
+    "  urllib.request.urlretrieve(\"https://www.csie.ntu.edu.tw/~cjlin/libsvmtools/datasets/binary/ijcnn1.tr.bz2\",\n",
+    "                             f\"{os.path.join(CLASSIFICATION_PATH, 'ijcnn1.tr.bz2')}\")\n",
+    "  for filepath in os.listdir(CLASSIFICATION_PATH):\n",
+    "    if filepath[-3:] == \"bz2\":\n",
+    "      abs_filepath = os.path.join(CLASSIFICATION_PATH, filepath)\n",
+    "      with bz2.BZ2File(abs_filepath) as fr, open(abs_filepath[:-4],\"wb\") as fw:\n",
+    "        shutil.copyfileobj(fr,fw)\n",
+    "\n",
+    "if not os.path.exists(REGRESSION_PATH):\n",
+    "  os.makedirs(REGRESSION_PATH)\n",
+    "  urllib.request.urlretrieve(\"https://www.csie.ntu.edu.tw/~cjlin/libsvmtools/datasets/regression/eunite2001\",\n",
+    "                              f\"{os.path.join(REGRESSION_PATH, 'eunite2001')}\")\n",
+    "  urllib.request.urlretrieve(\"https://www.csie.ntu.edu.tw/~cjlin/libsvmtools/datasets/regression/eunite2001.t\",\n",
+    "                             f\"{os.path.join(REGRESSION_PATH, 'eunite2001.t')}\")\n",
+    "  urllib.request.urlretrieve(\"https://www.csie.ntu.edu.tw/~cjlin/libsvmtools/datasets/regression/YearPredictionMSD.bz2\",\n",
+    "                             f\"{os.path.join(REGRESSION_PATH, 'YearPredictionMSD.bz2')}\")\n",
+    "  urllib.request.urlretrieve(\"https://www.csie.ntu.edu.tw/~cjlin/libsvmtools/datasets/regression/YearPredictionMSD.t.bz2\",\n",
+    "                             f\"{os.path.join(REGRESSION_PATH, 'YearPredictionMSD.t.bz2')}\")\n",
+    "  for filepath in os.listdir(REGRESSION_PATH):\n",
+    "    if filepath[-3:] == \"bz2\":\n",
+    "      abs_filepath = os.path.join(REGRESSION_PATH, filepath)\n",
+    "      with bz2.BZ2File(abs_filepath) as fr, open(abs_filepath[:-4],\"wb\") as fw:\n",
+    "        shutil.copyfileobj(fr,fw)\n",
+    "\n",
     "\n",
     "# Add to sys so we can import .py files.\n",
-    "sys.path.append(GOOGLE_DRIVE_PATH)\n",
     "!nvidia-smi\n",
+    "!pip install uvicorn uvicorn fastapi requests starlette\n",
```

Comment:
@charlesbvll is this necessary after we made those dependencies optional?

Reply:
No this is not necessary, this was just to be able to get it working before the optional dependencies PR is merged. I will probably remove that line once the other PR is merged. I added so it could be merged in any order.